compiler/AST/build.cpp

```diff
@@ -567,6 +567,46 @@ BlockStmt* buildImportStmt(Expr* mod, const char* rename, bool privateImport) {
   return buildChapelStmt(newImport);
 }
 
+//
+// Build an 'import' statement
+//
+BlockStmt* buildImportStmt(Expr* mod, std::vector<PotentialRename*>* names,
+                           bool privateImport) {
+  std::vector<const char*> namesList;
+
+  // Iterate through the list of names for unqualified access
+  for_vector(PotentialRename, listElem, *names) {
+    switch (listElem->tag) {
+      case PotentialRename::SINGLE:
+        if (UnresolvedSymExpr* name = toUnresolvedSymExpr(listElem->elem)) {
+          if (!privateImport) {
+            USR_FATAL(name, "unable to apply 'public' to this style of "
+                      "'import' at this time");
+          }
+
+          namesList.push_back(name->unresolved);
+        } else {
+          USR_FATAL(listElem->elem, "incorrect expression in 'import' for "
+                    "unqualified access, identifier expected");
+        }
+        break;
+      case PotentialRename::DOUBLE:
+        std::pair<Expr*, Expr*>* elem = listElem->renamed;
+        USR_FATAL(elem->first, "cannot rename symbols 'import'ed for "
+                  "unqualified access");
```

Comment:
I don't see a test of this (but might have missed it). Is it for the case like `import M.{x as y}` ? Should such syntax even be allowed?

Reply:
No, I meant to add one but I didn't (will do that now).  We do intend to support renaming in this case, I just didn't think doing it in this change was a good idea.